File: src/local.gwiasda.net/net.gwiasda.fima.bookings/Model/Booking.cs
﻿namespace Net.Gwiasda
{
    public class Booking
    {
        public Guid Id { get; set; } = Guid.NewGuid();
        public DateTime Timestamp { get; set; } = DateTime.Now;
        public string Text { get; set; } = string.Empty;
        public Guid CategoryId { get; set; } = Guid.Empty;
        public bool IsCost { get; set; }
        public decimal Amount { get; set; }
    }
}

File: src/local.gwiasda.net/net.gwiasda.local.ui/ClientApp/src/app/fima/fima-booking/booking.ts
export class Booking {
  id: string = '';
  timestamp: Date = new Date();
  text: string = '';
  categoryId: string = '';
  isCost: boolean = false;
  amount: number = 0;
  recurringType: string = 'einmalig';
  endDate: Date | null = null;
}
